src/components/00-materials/README.md

```diff
@@ -88,6 +83,6 @@ this.render() {
 2. Remove unnecessary code inside the SVG:
    - a. `data-name="Layer 2"` # layer info for vector drawing software
    - b. `<path fill="#fff" d="M0 0h96v96H0z"/>` # outer bounding box filled with white background
-3. - a. use the internal [svgo](https://github.com/svg/svgo)-invoking scripts `npm run build-icons` or `npm run build-images` to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `icons-raw` or `images-raw`
+3. - a. use the internal [svgo](https://github.com/svg/svgo)-invoking scripts `npm run build-icons` or `npm run build-images` from materials itself to clean up and optimize the SVGs, then copy the optimized files back from the `.tmp` folder to `icons-raw` or `images-raw`
```

Comment:
Now nowhere `build` script is described. This script is important to get the js-files @ icons/images folder.

Reply:
Yes that's also my opinion. The whole readme should be updated in my opinion. The usage section is not up to date, a description of the component is missing etc...

Reply:
I'll make an issue, because that should not be a part of this issue

Reply:
ok

Reply:
#1684